### DISCUSSION TITLE
Style the suggestion window

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [sxyazi/yazi#482] [1. Q&A]
Is there a way to style the suggestion window ?
![image](https://github.com/sxyazi/yazi/assets/61395246/4dea70a0-f461-48c0-a648-aec3722dc85c)

**Comment by sxyazi** [comment]
Yeah like this:

https://github.com/sxyazi/yazi/blob/93c8d90a519490cf3119d16341bf52860a4d9358/yazi-config/preset/theme.toml#L111-L117

Docs here: https://yazi-rs.github.io/docs/configuration/theme#which

**Reply by LamprosPitsillos** [reply]
Great thanks, but it would be great if we could also control some of the padding ,margins , height etc some of the descriptions get cut due to spacing.

**Reply by sxyazi** [reply]
There's a related issue, https://github.com/sxyazi/yazi/issues/219. Maybe we can allow users to set the number of columns, like setting it to two columns.

Alternatively, reimplementing it through Lua could give users the ability to fully customize their own styles using UI plugins, although this would be more complex.

**Reply by LamprosPitsillos** [reply]
Is the later a milestone for 2.1.0 ?

**Reply by sxyazi** [reply]
Done by https://github.com/sxyazi/yazi/pull/571
